### PR TITLE
Add 1.13 chunk format for snapshots

### PR DESCRIPTION
**Pull request by Brokkonaut**
Fixes WORLDEDIT-3714

### CODE REVIEW

File: worldedit-core/src/main/java/com/sk89q/worldedit/world/storage/ChunkStore.java [hunk 1]
```diff
@@ -78,7 +79,32 @@ public static BlockVector2D toChunk(Vector position) {
...
+                }
+            }
+        }
+
+        if (tag == null) {
+            throw new ChunkStoreException("Missing root 'Level' tag");
+        }
+
+        int dataVersion = rootTag.getInt("DataVersion");
+        if (dataVersion >= 1519) { // data version of Minecraft 1.13
```
Comment (kenzierocks) [inline worldedit-core/src/main/java/com/sk89q/worldedit/world/storage/ChunkStore.java]: Perhaps extract `1519` to a constant named `DATA_VERSION_MC_1_13`?

Then you won't need the comment, and it'll be easier to read in-line.

Comment (Brokkonaut) [reply worldedit-core/src/main/java/com/sk89q/worldedit/world/storage/ChunkStore.java]: Ok I changed that.

**Comment by Brokkonaut** [comment]
Parts of the new format are documented here: https://minecraft.gamepedia.com/Chunk_format
For the interpretation of the BlockStates array I used the decompiled sources (class DataBits in CB)

I tested it with many large selections in my current 1.13 world and had no problems restoring things, so I think I got everything correct.

**Comment by me4502** [comment]
Thanks

**Pull request merged**